Automattic/sensei
# Make minimal border rules less specific to allow overrides

### Changes proposed in this Pull Request

* There was an issue in themes with no predefined name colors where the border color of the minimal module style would not be respected. It was caused by a too specific CSS rule of the background color.
* This PR makes the CSS rules for the minimal border less specific to allow overrides.

### Testing instructions

* Create an outline block with Storefront theme.
* Set the module block to minimal style and set a named color.
* Observe that the correct color is set in the frontend.
